packages/react/src/ComparisonTable/ComparisonTable.tsx

```diff
@@ -29,6 +29,7 @@ export const _ComparisonTable = forwardRef(
       className,
       heading,
       featuredColumn = 1,
+      featuredColumnAccessibleLabel = '(featured)',
```

Comment:
Could we apply the parenthesis internally please, and not leave to end user to customize in case they forget?

Reply:
👍 794484bd7f9779e9bda0d2697580567c9ba4eeef